plutus-scb/src/Cardano/Node/Mock.hs

```diff
@@ -110,7 +148,9 @@ runChainEffects txQueue blockQueue stateVar eff = do
         $ interpret (handleZoomedState T.followerState)
         $ CE.handleChain
         $ Chain.handleControlChain
+        $ contramapLog NodeServerFollowerMsg
```

Comment:
Hm, why couldn't we use this before where we had to do the big dance with the prisms?

Reply:
The differnce is that here we transform `LogMsg a` to `LogMsg b`. In the other place we had `LogMsg a` to `Writer b` and we had been using prisms for similar transformations before so I thought it would fit in like that